internal/config/node/node.go

```diff
@@ -41,6 +41,13 @@ func ValidateConfig() error {
 			activated: &cgroupHasMemorySwap,
 			fatal:     false,
 		},
+		{
+			name:      "cgroup v2",
+			init:      CgroupIsV2,
+			err:       &cgroupIsV2Err,
+			activated: nil,
```

Comment:
Is there any way to set an appropriate true/false value here?

Reply:
no, because it's internal to libpod. we'd have to keep our own boolean somehow, which I think would be clunky